src/careers/current-openings/_single.html

```diff
@@ -54,8 +63,9 @@
                 block__padded-top
                 block__border-top">
         <h2>Job Description</h2>
-        <p>Lorem ipsum...place description text here</p>
+        {{ career.description }}
 
+        {# TODO: duties and departments are currently in the description
```

Comment:
Maybe clarify what needs to be done here. Duties and departments need to be removed from the description in the CMS? Is that right?